### ISSUE TITLE
[Enhancement] Give full staking rewards (16.5 Oxen) the "Service Node" label

### ISSUE THREAD

**Issue by mdPlusPlus** [oxen-io/oxen-electron-gui-wallet#285]
If you are a Service Node operator and fully deliver the 15,000 Oxen stake by yourself, the staking rewards currently show up with the label "-". Unfortunately, none of the provided transaction type filters (Incoming, Outgoing, Pending, Miner, Service Node, Governance, Stake, Failed) shows these transactions, only the "All" filter does.

I propose giving these transactions a label other than "-". Ideally they would be tagged as "Service Node", since this is what all the other, non-fully-self-provided stakes show up with and it would give a nice summary of *all* your staking earnings.

**Comment by mdPlusPlus** [comment]
Of course this should also be the case for fractional rewards, when being the operator, but not providing the full stake yourself.

**Comment by darcys22** [comment]
Is this in regards to the CSV export? "Miner" was the classification for all rewards which we recently renamed to "Reward" in https://github.com/oxen-io/oxen-core/pull/1446.

I'm not sure that "-" is intentional in this case. Could you elaborate further?

**Comment by mdPlusPlus** [comment]
![oxen_wallet](https://user-images.githubusercontent.com/9272131/124753063-d75fd580-df28-11eb-9926-e0a0df1134a1.png)

The top three transactions are rewards from shared stakes where I'm only a contributor, the last one is from one of my Service Nodes where I'm the operator.
When I go to the "Miner" tab it's empty, when I go to "Service Node" only the contribution rewards show up. I don't have a tab "Reward" yet. I'm on the 1.6.0 AppImage.

**Comment by darcys22** [comment]
Yeah that doesn't seem right, thanks for raising this

**Comment by mdPlusPlus** [comment]
Has any work been put into this yet?

**Comment by jagerman** [comment]
> Has any work been put into this yet?

Indirectly, yes: rewards are now batched, which works quite differently.  Have you tried this on the latest version?

**Comment by mdPlusPlus** [comment]
Yes, the behaviour is still exactly the same. Batched rewards also get the "-" label.

Also, something I just noticed: If you unlock a stake from a SN, the transaction gets the "Failed" label, although it clearly isn't a failed transaction, but a successful one.
